libs/common/src/interfaces/connection.interface.ts

```diff
@@ -19,4 +19,16 @@ export interface IConnectionsListCount {
     lastPage: number;
     data: IConnectionItem[];
   }
+
+  export interface ICreateConnectionUrl {
```

Comment:
Why this interface is kept in libs and the other one is inside connection microservice?

Reply:
@vivekayanworks Sir, The interface is stored in the common libraries because I intend to use it in both the API Gateway and the Connection service. So as per our discussion, I've included it in the common libraries' interface file. The other interface, however, is exclusive to the Connection service, as I plan to use it only there.